web/src/main/java/service/AccountService.java

```diff
@@ -0,0 +1,15 @@
+package service;
+
+import model.Account;
+import model.User;
+
+public interface AccountService {
```

Comment:
It is a bad practice to use entity models in Service classes. For each of the models you have in the data model, create DTOs in a new package under service. These DTOs will be used in the Service layer and will be mapped to the Entity models before we call the repository methods. Read more on how to do this: https://www.baeldung.com/java-dto-pattern